### PR TITLE
RN 0.73 compatibility for Android

### PR DESCRIPTION
**Pull request by fobos531**
@dominicstop 

See: https://github.com/react-native-community/discussions-and-proposals/issues/671#issuecomment-1597386954

### CODE REVIEW

**Comment by nandorojo** [comment]
We're migrating this to expo modules so that should hopefully address this.

**Comment by fobos531** [comment]
> We're migrating this to expo modules so that should hopefully address this.

https://github.com/dominicstop/react-native-ios-context-menu/blob/master/android/build.gradle#L65

Yep, great stuff! Best of luck with the migration, looking forward to it! Closing this one until further notice.

**Comment by SudoPlz** [comment]
@nandorojo is there an article or a comment anywhere in this repo that discusses the move to expo modules? (I wonder what the reasons behind adding that dependency were).

We're not using expo in our react-native project and I'm wondering if it's worth the overhead of dealing with expo modules since the only library that requires it is this one.

Thanks for all your great work. 🙏

**Comment by nandorojo** [comment]
Hey, there are a number of reasons.

TLDR, updating native libraries without expo is so annoying and hard that most libraries go entirely unmaintained.

Expo made it super easy to support the new/old architectures together and use JSI, all while writing Swift directly. 

I understand that not everyone has expo installed; however, it's becoming increasingly easier for a vanilla RN app to simply install expo-modules into their app in order to use third party modules.

Ultimately, the maintenance burden of spending days working on config files and dealing with React Native's unreasonable breaking changes from one update to another just isn't worth it.

Conversely, working with expo modules makes a library actually maintainable, and updating RN versions becomes easy.

**Comment by SudoPlz** [comment]
@nandorojo got it, I appreciate your quick response, that makes sense.
Thank you 🙏
